muted-tests.yml

```diff
@@ -304,6 +304,9 @@ tests:
 - class: org.elasticsearch.cluster.coordination.NodeJoinExecutorTests
   method: testSuccess
   issue: https://github.com/elastic/elasticsearch/issues/119052
+- class: org.elasticsearch.aggregations.bucket.SearchCancellationIT
```

Comment:
@original-brownbear as we discussed I am muting this test for main - it was already muted on 8.x.